benchexec/tools/pichecker.py

```diff
@@ -0,0 +1,23 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.cpachecker as cpachecker
+
+
+class Tool(cpachecker.Tool):
+    """
+    Tool info for PIChecker.
+    URL: https://gitlab.com/Lapulatos/pichecker
+    """
+
+    REQUIRED_PATHS = list(cpachecker.Tool.REQUIRED_PATHS) + ["resources"]
+
+    def version(self, executable):
+        return "1.0"
```

Comment:
Hard-coding a version number in this module is not a good idea. If people run a newer version of your tool with BenchExec they will get a wrong version number in the output, which could be very confusing.

The best would be to add a parameter like `--version` to your tool and read and return the actual version number here. You can call the utility method `self._version_from_tool` to do this easily.

If this is not possible, please return an empty value.

Reply:
Thank you for your suggestion!